appveyor.yml

```diff
@@ -38,4 +39,5 @@ build: off
 test_script:
   - cd %APPVEYOR_BUILD_FOLDER%
   - inv -e deps
-  - inv -e test --race --profile --fail-on-fmt
+  - inv -e test --coverage --race --profile --fail-on-fmt
+  - codecov -f profile.cov
```

Comment:
Is there a way to also make sure we don't fail the pipeline on this?

Reply:
The command never fails (though there is a flag for that).